marketplace/home/templates/index.html

```diff
@@ -6,4 +6,5 @@ <h1>Welcome to the Marketplace!</h1>
     <p>Discover amazing products and connect with sellers.</p>
     <a href="products" class="btn btn-primary">Browse Products</a>
     <a href="addproduct" class="btn btn-primary">Add Product</a>
+    <a href="seller_form.html" class="btn btn-primary">seller_registartion</a>
```

Comment:
Don't you think the url is wrong here.

Reply:
here i try all url such as seller_form,/marketplace/users/templates/seller_form,seller_form.html but still not works

Reply:
Look at the comment below.